### ISSUE TITLE
Few bugs

### ISSUE THREAD

**Issue by grappas** [irusanov/ZenStates#6]
Hi.
Gear:
- ASUS X570 TUF GAMING PLUS
- R7 3800x
- BIOS 1407 (AGESA 1.0.0.4 AB)
- chipset driver from 20/04/07
I've set manually:
Single Core - 4,55 GHz
All Cores - 4,3
Idle - left as It were
and boost goes no more than 4,3 GHz, even on Single Core. It reduced Core VID significantly (temps while idle are reduced) but It trashed performance.
If I only could manually set VID per state It would be useful. Now I can get more performance by reducing Vcore and adjusting PBO limits in BIOS.
Uninstall bat file not working also. I had to disable service from services dialog.

**Comment by irusanov** [comment]
Yeah, unfortunately those don't work on 3000 series. P-States don't work either :(
I'm working on a new version and the old one won't be supported anymore.
I have a way of detecting PBO limits and would be able to set new limits with 3000 series.

You can get a preview here, although a lot of work is needed, but I'm back to the project and will try to finish it in the next month.

[ZenStates_2.0.0_debug_20200529.zip](https://github.com/irusanov/ZenStates/files/4699573/ZenStates_2.0.0_debug_20200529.zip)
